Geolocator/Geolocator/Geolocator.Plugin.Abstractions/IGeolocator.cs

```diff
@@ -67,7 +67,8 @@ public interface IGeolocator
         /// <param name="minTime">Time</param>
         /// <param name="minDistance">Distance</param>
         /// <param name="includeHeading">Include heading or not</param>
-        Task<bool> StartListeningAsync(int minTime, double minDistance, bool includeHeading = false);
+		/// <param name="energySettings">Optional energy settings (iOS only)</param>
+		Task<bool> StartListeningAsync(int minTime, double minDistance, bool includeHeading = false, EnergySettings energySettings = null);
```

Comment:
I'm not sure what you're referring to. There's a `param` comment for the `energySettings` parameter.